gcp/cluster.tf

```diff
@@ -61,6 +62,7 @@ resource "google_compute_instance" "monitor" {
   name         = "rp-monitor-${local.deployment_id}"
   tags         = ["rp-cluster", "tf-deployment-${local.deployment_id}"]
   machine_type = var.monitor_machine_type
+  zone         = var.availability_zone[0]
```

Comment:
Should this always be pinned to the first listed az? Not sure I understand why we do this here, but allow picking one of the multiple listed in var.availability_zone in other gcp resources